src/Form/Validation/TwoFactorCodeConstraint.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+
+/*
+ * This file is part of Packagist.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *     Nils Adermann <naderman@naderman.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Form\Validation;
```

Comment:
To follow the usual Symfony convention, the `Validation` should not be nested in the `Form` one